src/test/projectsCreate.spec.ts

```diff
@@ -0,0 +1,220 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+
+import * as api from "../api";
+import { createFirebaseProject, ParentResource, ParentResourceType } from "../projectsCreate";
+import * as pollUtils from "../operation-poller";
+import { mockAuth } from "./helpers";
+import { OraWrapper } from "../oraWrapper";
+
+const PROJECT_ID = "the-best-firebase-project";
+const PROJECT_NUMBER = "1234567890";
+const PROJECT_NAME = "The Best Project";
+const PARENT_RESOURCE: ParentResource = { id: "1111111111111", type: ParentResourceType.FOLDER };
+const OPERATION_RESOURCE_NAME_1 = "operations/cp.11111111111111111";
+const OPERATION_RESOURCE_NAME_2 = "operations/cp.22222222222222222";
+
+// TODO(caot): Removed when "Deferred Analytics" and "Deferred Location" are launched
+const TIME_ZONE = "America/Los_Angeles";
+const REGION_CODE = "US";
+const LOCATION_ID = "us-central";
+
+describe("FirebaseResourceManager", () => {
+  describe("createFirebaseProject", () => {
+    let sandbox: sinon.SinonSandbox;
+    let mockOraWrapper: sinon.SinonMock;
+    let apiRequestStub: sinon.SinonStub;
+    let pollOperationStub: sinon.SinonStub;
+    let expectedCalledStubs: sinon.SinonStub[];
+
+    beforeEach(() => {
+      sandbox = sinon.createSandbox();
+      mockAuth(sandbox);
+      mockOraWrapper = sandbox.mock(OraWrapper.prototype);
+      apiRequestStub = sandbox.stub(api, "request");
+      pollOperationStub = sandbox.stub(pollUtils, "pollOperation");
+      expectedCalledStubs = [];
+    });
+
+    afterEach(() => {
+      mockOraWrapper.verify();
+      expectedCalledStubs.forEach((stub) => expect(stub.calledOnce).to.be.true);
```

Comment:
Expecting something outside of the tests is an anti-pattern. These are pretty easy one-liners to include in the test bodies along side the other assertions and doing so will give better error messages (rather than saying things like "the afterEach function failed for Such and Such test").

Reply:
Got it, thanks!

Reply:
Done